site/docs/getting-started/Instantiation.tsx

```diff
@@ -33,10 +33,10 @@ function getDialectSpecificCodeSnippet(
 ): string {
   const driverNPMPackageName = getDriverNPMPackageNames(packageManager)[dialect]
   const dialectClassName = DIALECT_CLASS_NAMES[dialect]
-  const poolClassName = 'Pool'
+  const poolClassName = packageManager === 'deno' ? 'Pool' : '{ Pool }'
```

Comment:
This actually "breaks" `pool: new ${poolClassName}({` on L43.

Now it'll print:
```ts
  pool: new { Pool }({
```